grant-boilerplate/short-pitch-for-assessment.txt

```diff
@@ -0,0 +1,31 @@
+Over the past five years, Software Carpentry and its sibling
+organization Data Carpentry have run hundreds of two-day workshops on
+basic software skills for more than 10,000 scientists.  Anecdotal
+feedback has been overwhelmingly positive, but anecdotes aren't
+evidence.  While three small-scale studies of Software Carpentry's
+impact have been done, we still don't really know:
```

Comment:
This sentence is awkward. How about:

Three small-scale studies of Software Carpentry's impact have been conducted, but we still don't really know: